Dockerfile.rocm

```diff
@@ -1,13 +1,22 @@
 # default base image
 ARG BASE_IMAGE="rocm/pytorch:rocm6.1_ubuntu20.04_py3.9_pytorch_2.1.2"
 
-FROM $BASE_IMAGE
+ARG COMMON_WORKDIR=/app
+ARG BUILD_HIPBLASLT="1"
+ARG BUILD_RCCL="1"
+ARG BUILD_FA="1"
+ARG BUILD_CUPY="0"
+ARG BUILD_TRITON="1"
+
+# -----------------------
+# vLLM base image
+FROM $BASE_IMAGE AS base
 USER root
 
 # Import BASE_IMAGE arg from pre-FROM
 ARG BASE_IMAGE
 RUN echo "Base image is $BASE_IMAGE"
-
+ARG COMMON_WORKDIR
```

Comment:
Need a default value? Or else Line 34 won't work

Reply:
It's defined in line 4 at the top of the Dockerfile, this ARG just imports it into this build stage